### PR TITLE
fix:bugs

### PR DESCRIPTION
**Pull request by talha-101digital**
### Why

Please give a short description why these code changes are being made, by replacing this line.

Link to Jira ticket: <place here>

### ToDo

- [ ]
- [ ]
- [ ]

If this is a `Draft:` or `WIP:` PR, please describe still needed actions, else delete section

### CODE REVIEW

File: src/components/bnpl-confirm.tsx [hunk 1]
```diff
@@ -213,13 +219,28 @@ return (
...
         />
       </View>
     </BottomSheet>
+    <BottomSheet isVisible={isBottomSheetOpen2}>
+      <View style={styles.wrapBottomSheet}>
+        <View style={styles.alertStyle}>
+        <AlertIcon size={60}/>
+        </View>
+        <Text style={styles.txtUpdate}>Sorry we are unable to offer you this financing product at the moment.</Text>
+        <Text style={styles.txtCopy}>We are always striving to improve our offerings and hope to be able to serve you soon.</Text>
```
Comment (sharmal33) [inline src/components/bnpl-confirm.tsx:234]: plz use translation file

File: src/components/review-final.tsx [hunk 2]
```diff
@@ -136,8 +143,14 @@ const ReviewFinalComponent: React.FC<IReviewDetails> = (props: IReviewDetails) =
...
     getUserProfile();
+    getState()
   }, [isFocused, formChange]);
-
+  useEffect(() => {
+    if(form.postcode.length > 0){
+      getCity(form?.postcode);
+    }
+  }, [form])
+  console.log('statedata', getCityData)
```
Comment (sharmal33) [inline src/components/review-final.tsx:153]: remove this

File: src/contexts/loan-component-context.tsx [hunk 3]
```diff
@@ -271,6 +273,24 @@ export const useLoanOriginationContextValue = (): ILoanContext => {
     } finally {
     }
   }, []);
+  const getState = useCallback(async () => {
+    try {
+      const response = await loanService.getState();
+      setStateData(response?.data);
+    } catch (e: any) {
+      console.log('e', JSON.stringify(e?.message));
```
Comment (sharmal33) [inline src/contexts/loan-component-context.tsx:281]: why this ?

File: src/contexts/loan-component-context.tsx [hunk 4]
```diff
@@ -271,6 +273,24 @@ export const useLoanOriginationContextValue = (): ILoanContext => {
...
+    } finally {
+    }
+  }, []);
+  const getCity = useCallback(async (id: string) => {
+    try {
+      const response = await loanService.getCity(id);
+      setCityData(response?.data);
+    } catch (e: any) {
+      console.log('e', JSON.stringify(e?.message));
```
Comment (sharmal33) [inline src/contexts/loan-component-context.tsx:290]: why this ?

**Review by sharmal33 — CHANGES_REQUESTED** [review]

File: src/contexts/loan-component-context.tsx (hunk 3 above)
Comment (talha-101digital) [reply src/contexts/loan-component-context.tsx:281]: to show city dropdown in addition info

File: src/contexts/loan-component-context.tsx (hunk 4 above)
Comment (talha-101digital) [reply src/contexts/loan-component-context.tsx:290]: to show city dropdown in addition info

**Review by sharmal33 — APPROVED** [review]

**Pull request merged**